tests/dev.rs

```diff
@@ -108,3 +108,138 @@ fn test_wukong_dev_help() {
 //
 //     temp.close().unwrap();
 // }
+
```

Comment:
I don't understand. Why all the comments in this file ?

Reply:
Initially it was because there are dynamic values in the output, so the snapshot will failed for each text run. I just found a way to solve this. Fixed.